### PR TITLE
feat: Onboard CDC Places Dataset

### PR DESCRIPTION
**Pull request by varshika06**
Onboard Pipelines:

1) datasets/cdc_places.local_data_for_better_health_county_data

Based on #<ISSUE-NUMBER>

Note: It's recommended to open an issue first for context and discussion.

## Checklist

Note: Delete items below that aren't applicable to your pull request.

- [x] Please merge this PR for me once it is approved.
- [x] If this PR adds or edits a dataset or pipeline, it was reviewed and approved by the Google Cloud Public Datasets team beforehand.
- [x] If this PR adds or edits a dataset or pipeline, I put all my code inside  datasets/cdc_places and nothing outside of that directory.
- [x] This PR is appropriately labeled.

### CODE REVIEW

File: datasets/cdc_places/_images/run_csv_transform_kub/csv_transform.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    df = df[headers]
+
+    logging.info(f"Transformation Process complete .. {source_file}")
+
+    logging.info(f"Saving to output file.. {target_file}")
+
+    try:
+        save_to_new_file(df, file_path=str(target_file))
+    except Exception as e:
+        logging.error(f"Error saving output file: {e}.")
```
Comment (adlersantos) [inline datasets/cdc_places/_images/run_csv_transform_kub/csv_transform.py:61]: I suggest we make the spacing consistent.

File: datasets/cdc_places/dataset.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - type: bigquery_dataset
+    # Google BigQuery dataset to namespace all tables managed by this folder
+    #
+    # Required Properties:
+    #   dataset_id
+    #
+    # Optional Properties:
+    #   friendly_name (A user-friendly name of the dataset)
+    #   description   (A user-friendly description of the dataset)
+    #   location      (The geographic location where the dataset should reside)
```
Comment (adlersantos) [inline datasets/cdc_places/dataset.yaml:56]: We can remove these

File: datasets/cdc_places/local_data_for_better_health_county_data/pipeline.yaml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    # Required Properties:
```
Comment (adlersantos) [inline datasets/cdc_places/local_data_for_better_health_county_data/pipeline.yaml:19]: Please remove all comments

**Review by adlersantos — CHANGES_REQUESTED** [review]
Looks good! Just some minor changes.

File: datasets/cdc_places/dataset.yaml (hunk 2 above)
Comment (varshika06) [reply datasets/cdc_places/dataset.yaml:56]: Done.

File: datasets/cdc_places/local_data_for_better_health_county_data/pipeline.yaml (hunk 3 above)
Comment (varshika06) [reply datasets/cdc_places/local_data_for_better_health_county_data/pipeline.yaml:19]: Done.

**Review by adlersantos — APPROVED** [review]

**Pull request merged**